### PR TITLE
feat: add commands for swebench

### PR DESCRIPTION
**Pull request by Sparkier**

### CODE REVIEW

**Comment by rbren** [comment]
Should this code be in the `./evaluation` dir? Why is it in codeact?

**Comment by Sparkier** [comment]
Yeah I was absolutely not sure where to put things and whether these commands should be just for the eval or something the agent should generally be augmented with.

File: agenthub/codeact_agent/commands.sh
Comment (xingyaoww) [inline agenthub/codeact_agent/commands.sh:1]: I am actually thinking about whether we should load these commands into the [sandbox's Dockerfile](https://github.com/OpenDevin/OpenDevin/blob/main/opendevin/sandbox/Dockerfile) so every different agent has access to these tools. @rbren what do you think?

File: agenthub/codeact_agent/codeact_agent.py [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+from opendevin.llm.llm import LLM
 from opendevin.observation import (
-    CmdOutputObservation,
     AgentMessageObservation,
+    CmdOutputObservation,
 )
+from opendevin.state import State
 
-from opendevin.llm.llm import LLM
+from .commands import parse_command_file
```
Comment (xingyaoww) [inline agenthub/codeact_agent/codeact_agent.py:18]: This `parse_command_file` basically provides a documentation for those bash commands. Maybe we can put it under `opendevin` so it is shared for every agent to use?

**Comment by rbren** [comment]
This is looking good to me! Maybe we should ask the eval folks?

**Review by xingyaoww — APPROVED** [review]
LGTM!

**Comment by xingyaoww** [comment]
I actually don't think evaluation is dependent on the agent / opendevin design - they only need one JSONL file that contains one patch output for each evaluation instances, so i think our current implementation should be fine?

**Comment by foragerr** [comment]
Are these functions meant for swe BENCH, as the title indicates?
or for swe AGENT as described in this issue: https://github.com/OpenDevin/OpenDevin/issues/570

**Comment by Sparkier** [comment]
> Are these functions meant for swe BENCH, as the title indicates? or for swe AGENT as described in this issue: #570

Well, these functions are akin to what the folks from SWE Agent did but are meant to improve performance on SWE Bench. So pick whichever you like. It also seems like we're thinking of applying this stuff more broadly even beyond both.

**Pull request merged**